virtual_rainforest/models/abiotic_simple/simple_regression.py

```diff
@@ -0,0 +1,588 @@
+r"""The ``models.abiotic_simple.simple_regression`` module uses linear regression
+and logarithmic interpolation to calculate atmospheric temperature and humidity profiles
+as a function of leaf area index and canopy layer height. The relationships are derived
+from :cite:t:`hardwick_relationship_2015` and :cite:t:`jucker_canopy_2018`.
+Soil temperature is interpolated between the surface layer and the air
+temperature at 1 m depth which equals the mean annual temperature.
+The module also provides a constant vertical profile of atmospheric pressure and
+:math:`\ce{CO2}`.
+Soil moisture and surface runoff are calculated with a simple bucket model based on
+:cite:t:`davis_simple_2017`.
+"""  # noqa: D205, D415
+
+from typing import Dict, List, Tuple, Union
+
+import numpy as np
+import xarray as xr
+from xarray import DataArray
+
+from virtual_rainforest.core.data import Data
+from virtual_rainforest.core.logger import LOGGER
+
+MicroclimateGradients: Dict[str, float] = {
+    "air_temperature_gradient": -1.27,
+    "relative_humidity_gradient": 5.4,
+    "vapour_pressure_deficit_gradient": -252.24,
+}
+"""Gradients for linear regression to calculate air temperature, relative humidity, and
+vapour pressure deficit as a function of leaf area index from
+:cite:t:`hardwick_relationship_2015`.
+"""
+
+MicroclimateParameters: Dict[str, float] = {
+    "soil_moisture_capacity": 90,
+    "water_interception_factor": 0.1,
+    "saturation_vapour_pressure_factor1": 0.61078,
+    "saturation_vapour_pressure_factor2": 7.5,
+    "saturation_vapour_pressure_factor3": 237.3,
+}
+"""Parameters for simple abiotic regression model."""
+
+Bounds: Dict[str, float] = {
```

Comment:
What happens when these bounds are exceeded?

Reply:
When the bounds are exceeded, the value is reduced to the bound value, for example the maximum relative humidity is 100%, so if it gets higher than that, the value is set to 100; equally if it is below 0 it is set to 0. This does obviously not conserve energy or matter, I will add a comment on that and a TODO